constants/role.constant.js

```diff
@@ -151,27 +151,26 @@ function createAllSingularRoles() {
     const allRoutes = Constants.Routes.allRoutes;
     let roles = [];
 
-    // i is unique integer so that objectId is constant
-    var i = 1000000;
     for (let routeGroupKey in allRoutes) {
-        if (!allRoutes.hasOwnProperty(routeGroupKey)) {
+        if (!Object.prototype.hasOwnProperty.call(allRoutes, routeGroupKey)) {
```

Comment:
What's the advantage of this over `!allRoutes.hasOwnProperty(routeGroupKey)`?

Reply:
See https://eslint.org/docs/rules/no-prototype-builtins. Long story short, `hasOwnProperty` can be manipulated to be malicious.